### DISCUSSION TITLE
SFTPGo webui not loading

### DISCUSSION THREAD

**Discussion by DrSpaldo** [tteck/Proxmox#3211] [Q&A] [answered]
I recently installed the LXC script for SFTPGo. The nominated webui http://192.168.1.15:8080/web/admin doesn't load. I was wondering if I was missing something? Did I need to do any config changes?

**Comment by tteck** [comment]
Try http://192.168.1.15:8080/web/admin

**Reply by DrSpaldo** [reply]
> Try http://192.168.1.15:8080/web/admin

sorry @tteck , that is what I was trying. I am not sure what happened when I pasted it in above. That link doesn't work for me

**Comment by DrSpaldo** [comment, marked as answer]
Removed the container & reinstalled. It is working now.. go figure
